lib/src/components/NxFilterInput/NxFilterInput.scss

```diff
@@ -31,8 +31,3 @@
     margin-right: 8px;
   }
 }
-
-.nx-filter-input--empty .nx-text-input__box {
-  // empty filter inputs should be styled the same as pristine text inputs
```

Comment:
This is still true - we want the light colored border on filter inputs when they are empty.

Reply:
> This is still true - we want the light colored border on filter inputs when they are empty.

I double checked with Kaleido and @alexalyons126 on this and they mentioned that it should be the default border color (grey-50) even when it's empty

Reply:
> > I double checked with Kaleido and @alexalyons126 on this and they mentioned that it should be the default border color (grey-50) even when it's empty
> 
> I find that very surprising considering previous conversations. @alexalyons126 just created this behavior for filter inputs about a month an a half ago and I recall asking whether the removal of the pristine state styling would impact that, and she confirmed that it would not. Please double check.

I double checked with @alexalyons126 and it's confirmed that filter input shouldn't have a different border color when it's empty.